### PR TITLE
Skn 391 enable profile image editing

### PR DESCRIPTION
**Pull request by KenyerRamirez**
I deleted the user updating middleware for updating user data from it own profile.

### CODE REVIEW

**Review by Alexjes98 — APPROVED** [review]

File: src/routes/users.js [hunk 1]
```diff
@@ -13,7 +13,6 @@ router.get(
 router.post('/', userController.createUser);
 router.put(
     '/:id',
-    [hasPermissions('user-management')],
```
Comment (maacaro) [inline src/routes/users.js:16]: @KenyerRamirez shouldn't be this endpoint protected  I mean if the user was created, shouldn't be the user it self the one that could update its profile picture?
is there any user role for the regular users trying to update its profile?

**Review by maacaro — CHANGES_REQUESTED** [review]
@KenyerRamirez need clarification

File: src/routes/users.js (hunk 1 above)
Comment (KenyerRamirez) [reply src/routes/users.js:16]: Not only its picture, the other data too, I mean: name, description, contact, etc. You're right, the middleware protect the endpoint, but this middleware doesn't let the user update it own data, for that reason I deleted that line.

Comment (maacaro) [reply src/routes/users.js:16]: @PedroLabrador is there any permission for the same user I wont't like users profile picture be allow to updates without being the owner user itself

Comment (maacaro) [reply src/routes/users.js:16]: what I mean is shouldn't be something like:
hasPermissions('user-management') || hasPermissions('user-owner'),

**Pull request closed without merging**